src/platform/silabs/wifi/wf200/WifiInterfaceImpl.cpp

```diff
@@ -1185,3 +1185,11 @@ void wfx_cancel_scan(void)
     }
     scan_cb = nullptr;
 }
+
+sl_status_t InitWiFiStack(void)
+{
+    // TODO: This function should include sl_wfx_hw_init() and sl_wfx_init() functions. Only done now to make MatterConfig platform
```

Comment:
Please create a ticket to follow up on this.

Reply:
created and added in comments